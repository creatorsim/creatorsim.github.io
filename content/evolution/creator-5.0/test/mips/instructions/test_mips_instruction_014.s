
#
# Creator (https://creatorsim.github.io/creator/)
#

.data
    b1:		.byte 127

    .align 1
    h1:		.half 65535
    b2:		.space 1
    .align 1
    h2:		.space 2

.text
main:

    la $t0, b1
    la $t1, h1

    lbu $t2, 0($t0)
    lhu $t3, 0($t1)

    la $t4, b2
    sb $t5, 0($t4)

    la $t6, h2
    sh $t7, 0($t6)
