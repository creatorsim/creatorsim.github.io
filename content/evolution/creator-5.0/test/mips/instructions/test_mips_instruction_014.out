
cr[PC]:0x40; ir[1,at]:0x200006; ir[8,t0]:0x200000; ir[9,t1]:0x200002; ir[10,t2]:0x7f; ir[11,t3]:0xffff; ir[12,t4]:0x200004; ir[14,t6]:0x200006; keyboard[0x0]:''; display[0x0]:''; 

